.travis.yml

```diff
@@ -16,7 +16,7 @@ before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 before_install:
-  - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
+  - openssl aes-256-cbc -K "$encrypted_5ffc1dec8cd8_key" -iv "$encrypted_5ffc1dec8cd8_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
```

Comment:
I think it's ok. Saw a bunch of hex and the word "encrypted" and wanted to make sure it wasn't accidentally leaking key material. But actually it's just a variable name.